src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudSpec.groovy

```diff
@@ -960,29 +942,21 @@ types .* or update switch properties and add needed encapsulation type./).matche
         and: "Flow rules are removed from the old dst switch"
         def flowInfoFromDb3 = database.getFlow(flow.flowId)
         wait(RULES_DELETION_TIME) {
-            with(northbound.getSwitchRules(dstSwitch.dpId).flowEntries.findAll {
-                !new Cookie(it.cookie).serviceFlag
-            }) { rules ->
-                rules.findAll {
-                    it.cookie in [flowInfoFromDb2.forwardPath.cookie.value, flowInfoFromDb2.reversePath.cookie.value]
-                }.empty
-            }
+            def cookies = northbound.getSwitchRules(dstSwitch.dpId).flowEntries
```

Comment:
IIRC we have class to work with switch rules (`org.openkilda.functionaltests.helpers.model.SwitchRules`). Let's re-use/extend it.